servers/express.js

```diff
@@ -1,56 +1,145 @@
 'use strict';
 
+const fsp = require('fs/promises');
+const crypto = require('crypto');
 const path = require('path');
+
+const bodyParser = require('body-parser');
 const express = require('express');
+const multer = require('multer');
 
 const Skeleton = require('./skeleton');
 const watcher = require('../file-watcher.js');
 
+// multer configs
+const storage = multer.diskStorage({
+  destination: './uploads/',
+  filename: function(req, file, cb) {
+    const basename = crypto.randomBytes(16).toString('hex');
+    const ext = path.extname(file.originalname);
+    cb(null, `${basename}${ext}`);
+  }
+});
+const upload = multer({storage: storage});
+
+// minimist configs
 const argvOptions = {
   alias: {
-    l: 'logfile',
+    l: 'logfile'
   },
   default: {
-    l: 'route-metrics.log',
+    l: ''
   },
 };
 const argv = require('minimist')(process.argv.slice(2), argvOptions);
 
-// the datarows
+// datarows and globals
+let firstTs, lastTs, version;
 const eventloopDataRows = [];
 const memoryDataRows = [];
 const cpuDataRows = [];
-let version;
 
 // the app
-const pathToLogFile = argv.logfile;
+let pathToLogFile = argv.logfile;
+let uploadedFiles = [];
 const app = express();
 
-app.all('/*', function (req, res, next) {
+// create the routers
+const clientRoutes = express.Router();
```

Comment:
this is a nice way to break things up!

i wouldn't do it now, but it's getting near the time when it will make sense to split the clientRoutes and apiRoutes into different files (it makes the code simpler to keep separate and it isolates one from the other).